### DISCUSSION TITLE
How do we request a site to scrape ?

### DISCUSSION THREAD

**Discussion by DiscusZ** [mealie-recipes/mealie#1707] [General]
I follow a woman who does Keto and make alot of her recipes.   Mealie will not import her stuff , just the image comes in that is it.

here is an example link:

https://www.lowcarblove.com/blog/butter-chicken-with-garlic-naan

**Comment by michael-genson** [comment]
You would need to file a feature request with the recipe scrapers library: https://github.com/hhursev/recipe-scrapers, but it doesn't look like the site includes the necessary recipe JSON

Alternatively, the new OpenAI integration can scrape recipes that don't have the supported JSON
